tests/system/Router/RouterTest.php

```diff
@@ -212,20 +212,28 @@ public function testAutoRouteFindsControllerWithFileAndMethod(): void
         $this->collection->setAutoRoute(true);
         $router = new Router($this->collection, $this->request);
 
-        $router->autoRoute('myController/someMethod');
+        copy(TESTPATH . 'system/Router/Controllers/MyController.php', APPPATH . 'Controllers/Mycontroller.php');
 
-        $this->assertSame('MyController', $router->controllerName());
-        $this->assertSame('someMethod', $router->methodName());
+        $router->autoRoute('Mycontroller/getSomemethod');
```

Comment:
```suggestion
        $router->autoRoute('mycontroller/getSomemethod');
```

> Important
The file must be called Helloworld.php, with a capital H. When you use Auto Routing, Controller class names MUST start with an uppercase letter and ONLY the first character can be uppercase.
...
Now visit your site using a URL similar to this:
example.com/index.php/helloworld

https://codeigniter4.github.io/CodeIgniter4/incoming/controllers.html#let-s-try-it-hello-world-legacy